specs/deneb/p2p-interface.md

```diff
@@ -65,34 +91,32 @@ The new topics along with the type of the `data` field of a gossipsub message ar
+Deneb introduces new global topics for blob sidecars.
 
 ##### `beacon_block`
 
-This topic is deprecated and clients **MUST NOT** expose in their topic set to any peer. Implementers do not need to do
-anything beyond simply skip implementation, and it is explicitly called out as it is a departure from previous versioning
-of this topic.
+The *type* of the payload of this topic changes to the (modified) `SignedBeaconBlock` found in deneb.
 
-Refer to [the section below](#transitioning-the-gossip) for details on how to transition the gossip.
+##### `blob_sidecar_{index}`
 
-##### `beacon_block_and_blobs_sidecar`
+This topic is used to propagate signed blob sidecars, one for each sidecar index. The number of indices is defined by `MAX_BLOBS_PER_BLOCK`.
 
-This topic is used to propagate new signed and coupled beacon blocks and blobs sidecars to all nodes on the networks.
+The following validations MUST pass before forwarding the `sidecar` on the network, assuming the alias `sidecar = signed_blob_sidecar.message`:
 
-In addition to the gossip validations for the `beacon_block` topic from prior specifications, the following validations MUST pass before forwarding the `signed_beacon_block_and_blobs_sidecar` on the network.
-Alias `signed_beacon_block = signed_beacon_block_and_blobs_sidecar.beacon_block`, `block = signed_beacon_block.message`, `execution_payload = block.body.execution_payload`.
-- _[REJECT]_ The KZG commitments correspond to the versioned hashes in the transactions list
-  -- i.e. `verify_kzg_commitments_against_transactions(block.body.execution_payload.transactions, block.body.blob_kzg_commitments)`
+- _[REJECT]_ The sidecar is for the correct topic -- i.e. `sidecar.index` matches the topic `{index}`.
+- _[IGNORE]_ The sidecar is not from a future slot (with a `MAXIMUM_GOSSIP_CLOCK_DISPARITY` allowance) -- i.e. validate that `sidecar.slot <= current_slot` (a client MAY queue future blocks for processing at the appropriate slot).
+- _[IGNORE]_ The sidecar is from a slot greater than the latest finalized slot -- i.e. validate that `sidecar.slot > compute_start_slot_at_epoch(state.finalized_checkpoint.epoch)`
+- _[IGNORE]_ The blob's block's parent (defined by `sidecar.block_parent_root`) has been seen (via both gossip and non-gossip sources) (a client MAY queue blocks for processing once the parent block is retrieved).
+- _[REJECT]_ The proposer signature, `signed_blob_sidecar.signature`, is valid with respect to the `sidecar.proposer_index` pubkey.
+- _[IGNORE]_ The sidecar is the only sidecar with valid signature received for the tuple `(sidecar.slot, sidecar.proposer_index, sidecar.index)`.
+  -- If full verification of the blob fails at a later processing stage, clients MUST clear the blob from this "seen" cache so as to allow a the valid blob to propagate. The next block producer MAY orphan the block if they have observed multiple blobs signed by the proposer for the same "seen" tuple.
```

Comment:
I understand now. It's using the proposer boost (and if low attestation weight so they know it will likely work).

Hmm, I'm okay putting this in here but maybe should merge the other orphaning spec.
Also the language probably needs to be "MAY attempt to orphan"

CC: @michaelsproul, the re-org god

Reply:
>  It's using the proposer boost 

I believe lighthouse calls it [proposer reorg](https://lighthouse-book.sigmaprime.io/late-block-re-orgs.html) which in some ways is the opposite of proposer boost: this does not appear in the spec anywhere but rather is a lighthouse-only feature as of now (afair) in which lighthouse validators penalize nodes that publish late blocks.

The open question here is whether we need the collaboration of other nodes to pull this off: do _they_ have to keep track of the duplicate blobs and "blacklist" the offending block as well or is it enough that the next proposer does so?

Reply:
Proposer re-orgs rely on the proposer-boost being larger than the weight of the attestations that the previous proposer managed to get (e.g. low because they released block late).

A few things
1. I don't think this really belongs in the networking spec at all. This is a fork-choice capability and a validator option
2. The obvious way to map that re-org functionality here is if the duplicate blobs reduce the ability of the block to garner attestations, leaving it re-orgable by proposer-boost weight at the next slot. I'm generally okay with making this into the fc/validator spec
3. Doing this in a coordinated fashion (rather than just by the leader in clear times that they can) seems much more dangerous and likely to lead to network splits, missed attestations, and other bad things. So I'm not really open to "everyone can try to re-org it" unless we have a very clear *how* and a clear analysis on the chances, the impact, the potential losses from honest folks, etc.

My preference for this PR would be to remove this line and move this convo into and issue

Reply:
https://github.com/ethereum/consensus-specs/issues/3261